apps/t_shirt_delivery/templates/registration/change_tshirt.html

```diff
@@ -103,21 +103,21 @@
 
             // Listen for button click
             newButton.addEventListener('click', () => {
```

Comment:
```javascript
newButton.addEventListener('click', () => {
...
}
```

should be 

```javascript
newButton.addEventListener('click', (event) => {
let btn = event.target;
...
}
```
But be better not use arrow function (if you do not use babel and transpile your ES6 code -> ES5). MS IE browser is [still supported ](https://docs.microsoft.com/en-us/lifecycle/announcements/internet-explorer-11-end-of-support) and [doesn't support arrow function](
https://kangax.github.io/compat-table/es6/ ).

should be

```javascript
newButton.addEventListener('click', function(event) {
let btn = event.target;
...
}
```